css/_uswds-theme-custom-styles.scss

```diff
@@ -2998,68 +2979,10 @@ $theme-table-column-gap: 4;
         $site-banner-link-color,
         $context: $banner-context
       );
-      @include u-pin("none");
-      display: inline;
-      margin-left: units(1);
-      position: relative;
-
-      &:after {
-        position: absolute;
-      }
-
-      &:hover {
-        // Underline added to inner text instead.
-        text-decoration: none;
-      }
-    }
-
-    &[aria-expanded="false"] {
-      background-image: none;
     }
 
     &[aria-expanded="true"] {
       background-image: none;
-
-      @include at-media-max("tablet") {
-        @include place-icon(
-          $banner-icon-close,
-          "after",
-          0,
-          middle,
-          "base-lighter"
-        );
-
-        &:before {
-          @include u-pin("y");
-          @include u-pin("right");
-          background-color: color("base-lighter");
-          content: "";
-          display: block;
-          height: units($size-touch-target);
-          width: units($size-touch-target);
-        }
-        &:after {
-          @include u-pin("y");
-          @include u-pin("right");
-        }
-      }
-
-      @include at-media("tablet") {
-        @include place-icon(
-          $banner-icon-chevron-up,
-          "after",
-          "2px",
-          middle,
-          $site-banner-background-color
-        );
```

Comment:
**Issue:** Removing these lines causes a visual regression where the up chevron does not show while the banner is open:

![banner-regression](https://github.com/uswds/uswds-site/assets/25211650/5b30c16f-06b3-4c19-bfb9-2e385a25fda5)

**Suggestion:** Restore the `at-media("tablet")` and `place-icon` mixins

> [!TIP]
> The reason we need to include this is because lines 2969-2976 override USWDS to properly set the closed banner icon with correct color contrast.
>
> https://github.com/uswds/uswds-site/blob/e085c99a87c97972c4bddcd9752c441ad9ee6eda/css/_uswds-theme-custom-styles.scss#L2969-L2976

Reply:
Good catch! I replaced this style and the chevron should be working correctly.